exporters/trace/gcp_exporter/internal/gcp_exporter_benchmark.cc

```diff
@@ -0,0 +1,229 @@
+#include <benchmark/benchmark.h>
+#include "exporters/trace/gcp_exporter/gcp_exporter.h"
+#include "opentelemetry/nostd/span.h"
+
+namespace cloudtrace_v2 = google::devtools::cloudtrace::v2;
+
+// These constants affect the overall runtime of the benchmark tests
+constexpr int kNumSpans = 100;
+constexpr int kNumIntAttributes = 30;
+constexpr int kNumStrAttributes = 30;
+constexpr int kNumBoolAttributes = 30;
+constexpr int kNumIterations = 1000;
```

Comment:
Is there any reason you want to set this manually (instead of letting the benchmark library take care of it)?